tsl/src/fdw/relinfo.c

```diff
@@ -78,6 +78,13 @@ apply_fdw_and_server_options(TsFdwRelInfo *fpinfo)
 								option_extract_extension_list(defGetString(def), false));
 			else if (strcmp(def->defname, "fetch_size") == 0)
 				fpinfo->fetch_size = strtol(defGetString(def), NULL, 10);
+			else if (strcmp(def->defname, "join_reference_tables") == 0)
```

Comment:
In the name of the parameter, I'd  draw attention to the fact that this is for replicated reference tables. E.g. `replicated_reference_tables`. Reference tables are not necessarily replicated, although we came to imply this in our terminology, so it might be confusing for the new users.

Reply:
A good point. I started a thread in Slack regarding the name.

Reply:
Renamed according to the discussion in Slack.